File: Spark学习/SparkSQL/src/scala/DataFrameCreate.scala
package cn.spark.study.sql

import org.apache.spark.SparkConf
import org.apache.spark.SparkContext
import org.apache.spark.sql.SQLContext

object DataFrameCreate {
  def main(args: Array[String]){
    val conf = new SparkConf()
    .setAppName("DataFrameCreate")
    
    val sc = new SparkContext(conf)
    val sqlContext = new SQLContext(sc)
    
    val df = sqlContext.read.json("hdfs://spark1:9000/student.json")
    
    df.show();
    
  }
}

File: Spark学习/SparkSQL/src/java/DataFrameCreate.java
package cn.spark.study.sql;

import org.apache.spark.SparkConf;
import org.apache.spark.api.java.JavaSparkContext;
import org.apache.spark.sql.DataFrame;
import org.apache.spark.sql.SQLContext;

public class DataFrameCreate {
	public static void main(String[] args) {
		SparkConf conf = new SparkConf()
				.setAppName("DataFrameCreate");
		
		JavaSparkContext sc = new JavaSparkContext(conf);
		SQLContext sqlcontext = new SQLContext(sc);
		
		DataFrame df = sqlcontext.read().json("hdfs://spark1:9000/students.json");
		
		df.show();
	}
}
